src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -143,6 +209,22 @@ boot_capmgr_mem_alloc(void)
 	cos_meminfo_alloc(capmgr_info, BOOT_MEM_KM_BASE, mem_sz);
 }
 
+static void
+boot_fault_handler_sinv_alloc(spdid_t spdid)
+{
+	invtoken_t  token = (invtoken_t)spdid;
+
+	struct cos_compinfo *comp_info = boot_spd_compinfo_get(spdid);
+	struct cos_compinfo *boot_info = boot_spd_compinfo_curr_get();
+
+	if (call_cap_op(comp_info->captbl_cap, CAPTBL_OP_SINVACTIVATE, BOOT_CAPTBL_FLT_DIVZERO, boot_info->comp_cap, (vaddr_t)fh_div_by_zero_err_inv, token)) BUG();
```

Comment:
These should not be "raw" calls using `call_cap_op`.  They should be part of `cos_kernel_api`.  Basically, `sinv_activate`.  I'm not sure if we even need any changes to that....